### ISSUE TITLE
Warn about classes or interfaces without accessibility specified

### ISSUE THREAD

**Issue by aarondandy** [aarondandy/csharp-analysis-tool#10]
A class or interface with no accessibility explicitly specified should be flagged with a warning
